src/test/resources/data/AutomatedSessionRemindersTest.json

```diff
@@ -22,7 +22,7 @@
       "googleId": "AutSessRem.instructor",
       "courseId": "AutSessRem.course",
       "name": "Test Ins for Aut Sessions Reminder",
-      "email": "${support.email}",
+      "email": "${test.instructor}@gmail.com",
```

Comment:
Putting the explanation here just in case anyone refers back to this PR in the future.

I am NOT trying to make the concept of support email address obsolete here, because the support email address is of course of great importance. What I am intending to make obsolete is its placeholder, i.e the `${support.email}`.

As far as this test is concerned, neither the original nor the new one is wrong, because we only want to check our chosen email box(es) to see if the session reminder emails are sent, but the original choice required us to have `${support.email}`, and that placeholder comes bundled with `.replace("${support.email}", Config.SUPPORT_EMAIL)` which is going to be applied to **ALL** UI test comparison, and that's highly redundant especially when _only one_ test uses it (I have verified this).

Which leads to the next question: why did we specifically use support email here? I did not manage to find an answer to that, because replacing it with any of the test accounts does not in any way affect the strength of the test, it only changes which email box(es) we should look the reminder emails in. With that in mind, I am confident that the placeholder `${support.email}` can be removed once and for all and here we go.